src/editorManager.js

```diff
@@ -402,7 +402,7 @@ class EditorManager {
 
       case KEY_CODES.BACKSPACE:
       case KEY_CODES.DELETE:
-        this.instance.emptySelectedCells();
+        this.instance.emptySelectedCells('delete');
```

Comment:
I think we can leave it at it is, `delete` we process as just `edit` action.
```suggestion
        this.instance.emptySelectedCells();
```